scripts/get-tag-name.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eo pipefail
+
+ROOT=$( cd `dirname $0`/..;  pwd )
+
+if [ "$#" -ne 3 ]; then
+    echo "$0 <PROJECT_NAME>"
+    exit 1
+fi
+PROJECT_NAME="$1"
+cd "$ROOT/$PROJECT_NAME"
+
+GET_VERSION_PY='import json,sys;print json.load(sys.stdin)["packages"][0]["version"]'
```

Comment:
Does this work OK?

I'm trying from the command line and getting an error:
```
$ cargo metadata --format-version 1 | python -c "$GET_VERSION_PY"
  File "<string>", line 1
    import json, sys; print json.load(sys.stdin)["packages"][0]["version"]
                            ^
SyntaxError: invalid syntax
```

Reply:
that is python2 syntax. You probably have python3. In which case it needs to be changed to 
`import json,sys;print(json.load(sys.stdin)["packages"][0]["version"])`

Reply:
I'll change it so that it works for both versions of python

Reply:
Thanks! The new command works.